.github/workflows/osx.yml

```diff
@@ -30,4 +30,4 @@ jobs:
     - name: Test Mathics
       run: |
         pip install pytest pexpect
-        make -j3 check
```

Comment:
I understand the pros and cons of `-j3`: in the normal case this saves time which I appreciate. But when there is an error, it make things harder to track down. 

There might be a way to set this in an individual run for debugging. I don't know.